lib/floe.rb

```diff
@@ -55,17 +53,4 @@ def self.logger
   def self.logger=(logger)
     @logger = logger
   end
-
-  # Set the runner to use
-  #
-  # @example
-  #   Floe.set_runner "docker", kubernetes", {}
-  #   Floe.set_runner "docker", Floe::Workflow::Runner::Kubernetes.new({})
```

Comment:
can we keep this as the public interface?

Reply:
`Floe.set_runner` essentially moved to `Floe::Workflow::Runner.register_scheme`.

There are 2 "APIs" here that I intentionally broke apart. There is an API for a plugin author and what they need to do to register a plugin.  Separately, there's an end-user API where end users can set things like Loggers.  IMO, the `Floe::Workflow::Runner` (or perhaps just `Floe::Runner`) namespace should be for the former, and the `Floe` namespace should be for the latter.